src/store/toolMenu/index.ts

```diff
@@ -8,15 +8,16 @@ export interface ToolMenuState {
 }
 
 const initialState: ToolMenuState = {
-  selectedKeys: []
+  // This is the default state. If no config is given, then load all tools
+  selectedKeys: ['default']
```

Comment:
As far as I understand this state member contains (or should contain) the currently _selected_ keys in the tool menu, not the available ones.

Reply:
I guess I missunderstood this. I thought this were here for the available tools. I Will redo this part and create a new store entry for the tools that should render